internal/rpchelp/helpdescs_en_US.go

```diff
@@ -0,0 +1,399 @@
+// Copyright (c) 2015 Conformal Systems LLC <info@conformal.com>
+//
+// Permission to use, copy, modify, and distribute this software for any
+// purpose with or without fee is hereby granted, provided that the above
+// copyright notice and this permission notice appear in all copies.
+//
+// THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+// WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+// MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+// ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+// WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+// ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+// OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+
+//+build !generate
+
+package rpchelp
+
+var helpDescsEnUS = map[string]string{
+	// AddMultisigAddressCmd help.
+	"addmultisigaddress--synopsis": "Generates and imports a multisig address and redeeming script to the 'imported' account.",
+	"addmultisigaddress-account":   "Unused (all imported addresses belong to the imported account)",
+	"addmultisigaddress-keys":      "Pubkeys and/or pay-to-pubkey-hash addresses to partially control the multisig address",
+	"addmultisigaddress-nrequired": "The number of signatures required to redeem outputs paid to this address",
+	"addmultisigaddress--result0":  "The imported pay-to-script-hash address",
+
+	// CreateMultisigCmd help.
+	"createmultisig--synopsis": "Generate a multisig address and redeem script.",
+	"createmultisig-keys":      "Pubkeys and/or pay-to-pubkey-hash addresses to partially control the multisig address",
+	"createmultisig-nrequired": "The number of signatures required to redeem outputs paid to this address",
+
+	// CreateMultisigResult help.
+	"createmultisigresult-address":      "The generated pay-to-script-hash address",
+	"createmultisigresult-redeemScript": "The script required to redeem outputs paid to the multisig address",
+
+	// DumpPrivKeyCmd help.
+	"dumpprivkey--synopsis": "Returns the private key in WIF encoding that controls some wallet address.",
+	"dumpprivkey-address":   "The address to return a private key for",
+	"dumpprivkey--result0":  "The WIF-encoded private key",
+
+	// GetAccountCmd help.
+	"getaccount--synopsis": "Lookup the account name that some wallet address belongs to.",
+	"getaccount-address":   "The address to query the account for",
+	"getaccount--result0":  "The name of the account that 'address' belongs to",
+
+	// GetAccountAddressCmd help.
+	"getaccountaddress--synopsis": "Returns the most recent unused external payment address for an account.\n" +
```

Comment:
The address isn't technically unused.  It might have received funds to it already.